submissions/m-ruslan/html-css-popup/index.html

```diff
@@ -0,0 +1,194 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="app.css" />
+    <link rel="icon" type="image/png" href="favicon.png" />
+    <title>Popup</title>
+  </head>
+  <body>
+    <nav>
+      <div class="nav-container">
+        <div class="nav-el nav-el-text" tabindex="0">
```

Comment:
I hope better way to interact with user throw native tags as `a` or `button`.
It will be more clear and [a11y](https://habr.com/ru/post/470091/) code